lib/block_view_types.go

```diff
@@ -3805,6 +3805,11 @@ type GlobalParamsEntry struct {
 	// then the first bucket will be [1000, 1099], the second bucket will be [1100, 1209], the third bucket will
 	// be [1210, 1330], etc.
 	FeeBucketRateMultiplierBasisPoints uint64
+
+	// FailingTransactionBMFRateBasisPoints is the factor of the transaction fee that is used for the computation
+	// BMF. The value is expressed in basis points. For example a value of 2500 means that 25% of the fee will be
+	// failing transaction fee will be used in the BMF algorithm.
+	FailingTransactionBMFRateBasisPoints uint64
```

Comment:
Can we rename this to FailingTransactionBMFMultiplierBasisPoints? Rate makes me think of "fee rate" and it was confusing until I read the code.

Reply:
Just to clarify why I don't like "rate": Rate is usually something "per" something else. Like "per kilobyte." Meanwhile, multiplier is just multiplied by something else, which is what this is.

Reply:
Yeah, I like Multiplier more too